scm-ui/ui-components/src/apiclient.ts

```diff
@@ -124,6 +122,10 @@ export function createUrl(url: string) {
   return `${contextPath}/api/v2${urlWithStartingSlash}`;
 }
 
+export function createUrlWithIdentifiers(url: string): string {
+  return createUrl(url) + "?X-SCM-Client=WUI&X-SCM-Session-ID=" + sessionId;
```

Comment:
Shouldn't we use constants here for `X-SCM-Client`, `WUI` and `X-SCM-Session-ID`?

Reply:
Okay, this isn't that easy. Forget about it.